Game/Source/GDKTestGyms/Private/LatencyTracer.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "LatencyTracer.h"
+
+#include "Async/Async.h"
+#include "Engine/World.h"
+#include "GeneralProjectSettings.h"
+#include "SpatialGDKSettings.h"
+
+DEFINE_LOG_CATEGORY(LogLatencyTracer);
+
+namespace LatencyTracerInternal
+{
+	FUserSpanId GDKSpanToUser(const FSpatialGDKSpanId& Span) // TODO: Expose the internal GDK versions of these
```

Comment:
Maybe out of scope but it might be nice to have this conversation in  FSpatialGDKSpanId itself. I think we do this conversation elsewhere in code in the same way.

Reply:
Is it worth removing these TODO's or exposing a way to convert between these types publicly? Maybe we could just inline the methods in a header somewhere